drivers/can/can_loopback.c

```diff
@@ -149,11 +151,39 @@ int can_loopback_configure(struct device *dev, enum can_mode mode,
 	return 0;
 }
 
+static enum can_state can_loopback_get_state(struct device *dev,
+					     struct can_bus_err_cnt *err_cnt)
+{
+	if (err_cnt) {
+		err_cnt->tx_err_cnt = 0;
+		err_cnt->rx_err_cnt = 0;
+	}
+
+	return CAN_ERROR_ACTIVE;
+}
+
+#ifndef CONFIG_CAN_AUTO_BUS_OFF_RECOVERY
+int can_stm32_recover(struct device *dev, s32_t timeout)
+{
+	return 0;
```

Comment:
I think we should have here `ARG_UNUSED(dev);` and `ARG_UNUSED(timeout)` like we do in other part of the code base.